universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java

```diff
@@ -66,7 +68,7 @@ public HtmlBundle getBundle(HtmlBundle bundle) {
         meta().attr("name", "viewport").attr("content", "width=device-width, initial-scale=1"));
 
     // Add the warning toast, only for staging
-    if (STAGING_HOST_NAME.equals(hostName)) {
+    if (isStaging) {
```

Comment:
Optional: Why add this isStaging field if it is only used once? 

You could leave it as if(hostName.equals(stagingHostname)). I understand you might be going for clarity with isStaging field.

Reply:
`isStaging` seemed like it takes slightly less effort for the reader's eyes to parse when they read the method